### PR TITLE
Add page for setting a Custom Working Directory 

### LINKED ISSUE #450: Add example for setting the custom working directory
Unless vapor/vapor#2228 fixes this, we should have docs with images for setting the custom working directory

### PR DESCRIPTION
**Pull request by cbgrey**
Add a page with images explaining how to set a custom working directory in Xcode.
Fixes #450

### CODE REVIEW

File: 4.0/docs/custom-working-dir.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Setting a Custom Working Directory
+By default Xcode will reference a very obscure working directory called DerivedData for various assets which can cause significant problems or confusion. To avoid this, set a *Custom Working Directory* in the Xcode Scheme for your project. 
+
+You will notice it is *not* set when you see this warning in the output: 
+![Output Warning](../images/set-working-dir-01.png)
```
Comment (grundoon) [inline 4.0/docs/custom-working-dir.md:5]: The `images` directory is at the same level as this page itself, not in the parent directory, so the links are all broken as currently specified.

Comment (cbgrey) [reply 4.0/docs/custom-working-dir.md:5]: Hm. I built and viewed it before I did the PR but will review.

**Comment by cbgrey** [comment]
Good suggestions. For those new to Vapor it would be helpful to have this early in the docs so I'd go with your first recommendation. I'll create an "Xcode" section and include it there.

File: 4.0/docs/xcode.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Xcode
+
```
Comment (tanner0101) [inline 4.0/docs/xcode.md:2]: Something here to explain the section would be nice, like:

> This page goes over some tips and tricks for using Xcode. If you use a different development environment, you can skip this.

File: 4.0/docs/xcode.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+## Setting a Custom Working Directory
+By default Xcode will reference a very obscure working directory called DerivedData for various assets which can cause significant problems or confusion. To avoid this, set a *Custom Working Directory* in the Xcode Scheme for your project.
```
Comment (tanner0101) [inline 4.0/docs/xcode.md:4]: Headers should have a newline after them.

**Pull request merged**